src/finder.js

```diff
@@ -109,6 +111,15 @@ export function find (node, args, raw, minimist) {
   }
 }
 
-export function run ({command, args}) {
-  return command.handler.apply(this, args)
+export function run ({command, args}, root) {
+  if (typeof command.handler === 'function') {
+    return command.handler.apply(this, args)
+  } else if (typeof command.handler === 'string') {
+    return require(path.join(root, command.handler)).default.apply(this, args)
+  } else if (typeof command.handler === 'undefined') {
+    console.log(command)
+    return require(path.join(root, command.path)).default.apply(this, args)
+  }
+
+  throw new Error('?') // TODO
```

Comment:
- Throw an specific error for "handler is not a supported type, expected function, string or handler defined at `{{path}}`, found i.e. Number" (UnsupportedHandler?)
- We should surround the requires with `try/catch` and check for `Module not found`, rethrow specific error with info about where command should be found and wasn't. (HandlerNotFound?)